### PR TITLE
Added Exception catch and ignore for DBuser exists

### PR DESCRIPTION
**Pull request by hkveeranki**
In reference to #1428

### CODE REVIEW

File: lib/private/Setup/MySQL.php [hunk 1]
```diff
@@ -87,14 +87,17 @@ private function createDatabase($connection) {
...
+			$name = $this->dbUser;
+			$password = $this->dbPassword;
+			// we need to create 2 accounts, one for global use and one for local user. if we don't specify the local one,
+			// the anonymous user would take precedence when there is one.
+			$query = "CREATE USER '$name'@'localhost' IDENTIFIED BY '$password'";
+			$connection->executeUpdate($query);
+			$query = "CREATE USER '$name'@'%' IDENTIFIED BY '$password'";
+			$connection->executeUpdate($query);
+		}
+		catch (\Exception $ignore){}
```
Comment (nickvergessen) [inline lib/private/Setup/MySQL.php]: I guess we should log this, similar to the exceptions in `createDatabase`

**Review by nickvergessen — CHANGES_REQUESTED** [review]

**Comment by nickvergessen** [comment]
Also as per IRC you are missing the sign-off on your commit:
https://github.com/nextcloud/server/blob/master/.github/CONTRIBUTING.md#sign-your-work

**Comment by nickvergessen** [comment]
@MorrisJobke @LukasReschke

**Review by nickvergessen — APPROVED** [review]

**Comment by LukasReschke** [comment]
```
Doctrine\DBAL\Exception\DriverException: An exception occurred while executing 'UPDATE "oc_filecache" SET "storage" = ?, "path" = ?, "path_hash" = ? WHERE "fileid" = ?' with params [358, "files\/folder2\/folder1\/test.txt", "68142157b2aa9eb749de7b5dd59496cd", "2994"]:
382s
100
SQLSTATE[HY000]: General error: 13 database or disk is full
382s
102
```

/me looks at @MorrisJobke

**Comment by MorrisJobke** [comment]
> /me looks at @MorrisJobke

Damn ... I will check this.

**Comment by MorrisJobke** [comment]
> Damn ... I will check this.

Weird - I can't find anything suspicious.

**Pull request merged**